### DISCUSSION TITLE
Swagger integration

### DISCUSSION THREAD

**Discussion by mbnoimi** [serverpod/serverpod#127] [Q&A]
Is there any plan to support Swagger generation? 
In most cases API documentation is a vital feature for any backend.

**Comment by mbnoimi** [comment]
@vlidholt I know you're focusing on coding (I noticed many commits recently) but my question still without any answer. May you please take a look into it?

**Reply by mayurdhurpate** [reply]
Yes, would love to know more about it. Another related question here: https://www.reddit.com/r/FlutterDev/comments/t6i51p/comment/hzopt0x/?utm_source=share&utm_medium=web2x&context=3
